### ISSUE TITLE
OCTO-DE review of removing content from tables on birth defect compensation rates page

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#43548]
## Issue Description
As part of our work to make sure all VA.gov pages are 508 compliant, we've been removing content from tables that shouldn't be in tables and reworking the page. 

We took content on the birth defect compensation rates page out of tables in ticket #42774.

This ticket is for Beth's review of that work and any changes the content team needs to make. We'll also track stakeholder approval, the pre-publishing checklist and publishing in this ticket.

Preview link: http://preview-prod.vfs.va.gov/preview?nodeId=993

## Tasks
- [x] OCTO-DE review (Beth)
- [x] Send to stakeholder for review (Randi/Laura)
- [ ] Apply changes to past rates pages (Katherine)
- [x] Copyedit with pre-publishing checklist (Megan)
- [x] Publish pages (Megan)

## Acceptance Criteria
- [x] Page with changes is live on VA.gov

**Comment by bethpottsVADEPO** [comment]
@RLHecht @katherine-fung This reworking of the tables looks great. You can move forward with it. The only question I have is whether we keep the cents for each rate. I think we decided that if it's zero-zero for all rates in a block of content, we'd leave the decimal and zeros off. @megzehn can you confirm? 

Also, these changes are so straightforward that you don't need to run it past Elana in disability comp. 

Sorry it took me so long to circle back to the tables work, and thanks for your patience!

**Comment by megzehn** [comment]
Yes, that's correct, @bethpottsVADEPO! (Here's the [Slack conversation for reference, with an example.](https://dsva.slack.com/archives/C03MP8MEPCN/p1658973637587149))

**Comment by katherine-fung** [comment]
Thanks all! 

We're waiting for the new rates for 12/1/22 to be approved. Once they're approved, I'll change the rates and drop the decimals and zeros if the rates don't have cent amounts.

**Comment by RLHecht** [comment]
This work is completed now on the 2023 and 2022 past rates pages. @katherine-fung Do you want to make these same changes to the other past rates pages we have in Drupal?

**Comment by RLHecht** [comment]
As discussed we are not going to do this work on any past rates pages. So this is now complete. Closing ticket!
